src/supervisor3.erl

```diff
@@ -134,8 +134,10 @@
 %%--------------------------------------------------------------------------
 
 -ifdef(use_specs).
+-type tref() :: reference().
```

Comment:
`timer:tref().`

Reply:
Nej, bare erlang timer is used here, it returns reference: http://erlang.org/doc/man/erlang.html#send_after-4